src/screens/PasswordRecovery/componentes/EmailForm/index.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import {
+  KeyboardAvoidingView,
+  TextInput
+} from 'react-native'
+import * as Yup from 'yup'
+import { Formik } from 'formik'
+import { isEmpty, values } from 'lodash'
+import { Body, Description, GrayButton, PrimaryButton } from '~/components'
+import styles, { PLACEHOLDER_COLOR } from './styles'
+
+const initialValues = {
+  email: ''
+}
+
+const Error = ({ error }) => (
+  error && (
+    <Body style={styles.error}>
+      Email invalido
+    </Body>
+  )
+)
+
+const schema = Yup.object().shape({
+  email: Yup.string()
+    .email('Email inválido')
+    .required('Requerido')
+})
+
+const formHasBlankValues = (formValues) => {
+  const blankValues = values(formValues).filter(element => element === '')
+  return !isEmpty(blankValues)
+}
```

Comment:
There is a function in `mixins` for this one. It's not necessary to change it if this works, but the one in `mixins` has a unit test